qml/components/messageContent/MessageVideo.qml

```diff
@@ -26,18 +27,26 @@ import "../../js/debug.js" as Debug
 MessageContentBase {
     id: videoMessageComponent
 
-    property var videoData:  ( rawMessage.content['@type'] === "messageVideo" ) ?  rawMessage.content.video : ( ( rawMessage.content['@type'] === "messageAnimation" ) ? rawMessage.content.animation : rawMessage.content.video_note )
-    property string videoUrl;
-    property int previewFileId;
-    property int videoFileId;
-    property bool isVideoNote : false;
-    property bool fullscreen : false;
+    property bool isVideoNote : rawMessage.content['@type'] === "messageVideoNote";
+
+    property var contentBase: ( rawMessage.content['@type'] === "messageVideo" ) ?  rawMessage.content.video : ( ( rawMessage.content['@type'] === "messageAnimation" ) ? rawMessage.content.animation : rawMessage.content.video_note )
```

Comment:
Not that it really matters, but I would pass `content_type` role as a property to `MessageContentBase` to avoid these `content['@type']` lookups.